tests/manual_explosion.py

```diff
@@ -13,11 +13,11 @@
 start = time.time()
 #s = b.factory.blank_state(add_options={"COMPOSITE_SOLVER"})
 s = b.factory.blank_state(add_options={"COMPOSITE_SOLVER"}, remove_options={"LAZY_SOLVES"})
-pg = b.factory.path_group(s)
-angr.path_group.l.setLevel("DEBUG")
-pg.step(300)
+sm = b.factory.simgr(s)
+angr.simgr.l.setLevel("DEBUG")
```

Comment:
AttributeError: 'module' object has no attribute 'simgr'

(you want angr.manager. names are hard)

Reply:
Good catch! Thanks